tests/integration/testnet/testNetNetwork.spec.js

```diff
@@ -532,4 +531,207 @@ describe('Liquality wallet...', async () => {
   })
+  it('Send BTC to another Wrong address. check Review option has been disabled', async () => {
+    const importWithSeedOptionElement = await page.waitForSelector('#import_with_seed_phrase_option', {
+      visible: true
+    })
+    await importWithSeedOptionElement.click()
+    console.log('Import with seed phrase option has been displayed')
+    await page.waitForSelector('#import-wallet_top', {
+      visible: true
+    })
+    console.log('Import wallet page hase been loaded')
+    // Get the existing SEED words as environment variables
+    let words
+    const SEED_WORDS = process.env.SEED_WORDS
+    if (!SEED_WORDS) {
+      throw new Error('Please provide SEED_WORDS as environment variables')
+    } else {
+      words = SEED_WORDS.split(' ')
+    }
+
+    const seedsWordsCount = await page.$$('#import_wallet_word')
+    for (let i = 0; i < seedsWordsCount.length; i++) {
+      const wordInput = seedsWordsCount[i]
+      await wordInput.type(words[i])
+    }
+
+    // Click on continue button
+    await page.click('#import_wallet_continue_button')
+    console.log('Import wallet continue button has been clicked')
+
+    // Create a password
+    await page.type('#password', password)
+    await page.type('#confirmPassword', password)
+    await page.click('#next_button') // click on continue
+
+    // overview page
+    await page.waitForSelector('#overview', {
+      visible: true
+    })
+
+    await page.click('#head_network')
+    await page.waitForSelector('#testnet_network', {
+      visible: true
+    })
+    console.log('user successfully logged in after import wallet')
+
+    await page.click('#testnet_network')
+    const overviewText = await page.$eval('.text-muted', el => el.innerText)
+    expect(overviewText, 'Testnet overview header').contain('TESTNET')
+    console.log('user successfully changed to TESTNET')
+
+    // check Send & Swap & Receive options have been displayed
+    await page.waitForSelector('#send_action', {
+      visible: true
+    })
+    await page.waitForSelector('#swap_action', {
+      visible: true
+    })
+    await page.waitForSelector('#receive_action', {
+      visible: true
+    })
+
+    // validate the testnet asserts count
+    const assetsCount = await page.$eval('#total_assets', el => el.innerText)
+    expect(assetsCount, 'Total assets in TESTNET should be 6').contain('6 Assets')
+
+    // Check the currency
+    const walletStatText = await page.$eval('.wallet-stats', el => el.innerText)
+    expect(walletStatText, 'Wallet stats has currency should be USD').contain('USD')
+
+    // Check the Total amount
+    await page.waitForTimeout(10000)
+    const totalAmount = await page.$eval('.wallet-stats_total', el => (el.innerText).replace(/[.,\s]/g, ''))
+    expect(parseInt(totalAmount), 'Funds in my wallet should be greater than 2000 USD').greaterThanOrEqual(2000)
+    console.log(chalk.green('After Import wallet, the funds total greater than 2000 USD'))
+
+    // Click on SEND Option
```

Comment:
Everything above this seems to be outwith the scope of the current test,
If there's something that needs to be done before each test, it should be abstracted out into a utility to avoid too much duplication. In that way, if we update something to do with importing/password, we just need to update once. 
Do you agree with this pattern?

Reply:
Yes, @monokh  I have a separate task to put these tests in Page object pattern so that we will keep page actions in each Class then re use them across the test suite.

Reply:
Just for now Trying to add some coverage for the use cases, then i will break down these into each page object util